src/settings/MappingProfiles/detailsSections/view/HoldingsDetailsSections/HoldingsNotes.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  MultiColumnList,
+  NoValue,
+} from '@folio/stripes/components';
+
+import {
+  getContentData,
+  getFieldValue,
+  transformSubfieldsData,
+  getBooleanLabelId,
+  getValueById,
+} from '../../utils';
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import { mappingProfileFieldShape } from '../../../../../utils';
+
+import css from '../../../MappingProfiles.css';
+
+export const HoldingsNotes = ({ mappingDetails }) => {
+  const noValueElement = <NoValue />;
+
+  const notes = getFieldValue(mappingDetails, 'notes', 'subfields');
+
+  const notesVisibleColumns = ['noteType', 'note', 'staffOnly'];
+  const notesMapping = {
+    noteType: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.field.notes.noteType`} />
+    ),
+    note: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.field.notes.note`} />
+    ),
+    staffOnly: (
+      <FormattedMessage id={`${TRANSLATION_ID_PREFIX}.field.notes.staffOnly`} />
+    ),
```

Comment:
Here is not so long strings to split them, you can write each value in save string with each key

Reply:
Sorry, but it can't be written in one string due to eslint warnings